relay/relay-network.md

```diff
@@ -0,0 +1,67 @@
+# Relay Network
+
+Relay network is defined by the federation of relay servers exchanging messages to route from a publishing client to a subscribing client on the same topic.
+
+## Goals
+
+Relay servers will route messages through other servers in the network that also have matching subscribers on a topic whilst maintaing the same expected behavior described by a publisher when requested to its own server
+
+## Network Message
+
+An Network message is defined as a serialized message of a successfully cached message from a publishing request from a connected client with one of the servers.
+
+Context:
+
+- **topic** - a string defining the target topic for message subscription.
+- **hash** - a string defining the message id as the sha256 hash of itself.
+
+Header:
+
+- **prefix** - a string prefix ("irn") for exchanged iridium messages.
+- **version** - an integer number of the serialized schema for messages.
+- **tag** - an integer label that identifies a message without knowing its contents.
+- **prompt** - a flag that identifies whether the server should prompt a user action.
```

Comment:
Just wonder if the `prompt` flag is still necessary when we have tags. Could relay server decide wether to prompt or not  base on tag value now? Or we want to leave that control to the client and potential 3rd party network consumers?

Reply:
I think that the prompt is useful for triggering webhooks. Knowing when do to the webhook is something that the wallet knows and the relay can't know this. The effort on the echo server should clarify whether this is more or less useful.